src/shared/cgroup-setup.c

```diff
@@ -614,7 +614,7 @@ int cg_migrate(
                 if (r < 0)
                         return RET_GATHER(ret, r);
 
-                while ((r = cg_read_pid(f, &pid)) > 0) {
+                while ((r = cg_read_pid(f, &pid, 0)) > 0) {
```

Comment:
Here, is it safe to ignore zeros?? Not sure, but after this is called, that the 'from' cgroup is expected to be empty, no?

Reply:
Hm, maybe this function wants to read them instead, and return an error if it encounters one?
Since I'm pretty sure all the code below this would not take kindly to pid being zero.